packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -35,7 +36,12 @@ export function BasePostContent({
 
   return (
     <>
-      {customNavigation ?? (
+      {isPostPage ? (
+        <PostContentHeaderMobile
```

Comment:
As the component is called mobile, shouldn't we maybe make it dynamic and then only show if isMobile?
(Optional idea)

Reply:
Yeah, that makes sense, though, in the component itself, we have `tablet:hidden` - so it shouldn't show. I tried to use CSS over JS but we can make an exception here. Wdyt?

Reply:
Not sure how many other things it loads, so either way works for me 🙏
EG if that component would do a lot of computing i would say move it up, if it's mostly rendering don't worry about it.

Reply:
Ah, that is a good point about it. The component is fairly simple and doesn't do much, so I think it is safe to say we should be good with just having it hidden.